discussment-ws/docs/discussment-service-v1.yaml

```diff
@@ -0,0 +1,583 @@
+#%RAML 1.0
+title: Discussment web service
+version: v1
+baseUri: /{version}
+mediaType: application/json
+securitySchemes:
```

Comment:
Security schemes can be in separate file and only included here. Makes the API document more readable when editing. For reading full API html format is better imho.